python/GangaDirac/test/Unit/Utilities/TestDiracUtilities.py

```diff
@@ -12,6 +9,7 @@
 def config_files():
     """
     Load the config files in a way similar to a full Ganga session
+    NB: This is taken to be only within the context of __this__ file!
```

Comment:
This is a standard pytest fixture and the pytest framework defines the scope of it. I suggest reading up on pytest as their docs will explain it better than I can.